src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,55 @@
+.clock {
+  position: absolute;
+  width: $clock-width;
+  height: $clock-height;
+  border-radius: 50%;
+  background-color: rgb(210, 205, 205);
+
+  &__center {
+    position: relative;
```

Comment:
strange solution, why relative? shouldn't it be easier to use absolute here instead?